### ISSUE TITLE
Timer Job runs and shows success, but it only takes 7 seconds and nothing happens

### ISSUE THREAD

**Issue by whitenoiz** [Nauplius/FoundationSync#23]
We have staff photos in AD, and I am trying to get them to sync from Exchange 2010 into Sharepoint Foundation 2013 (latest patch versions on both). I have installed and configured foundation sync and the timer job seems to run successfully, however the UserPhotos library remains empty and nothing else happens.
Help gratefully appreciated!

**Comment by tseward** [comment]
Are you able to navigate to https://outlook.example.com/ews/Exchange.asmx/s/GetUserPhoto?email=validemailddr@domain.com&size=HR648x648 and get a valid picture for a user who has uploaded one?

**Comment by whitenoiz** [comment]
Hi Trevor,

We have only added 96 x 96 images to Active Directory so far.  Do we need
648 x 648 for this to work?

Steve.

**Comment by whitenoiz** [comment]
Hi Trevor,

I added a 648x648 image to AD on my account and tried to retrieve it via
the URL you sent (modified without details), but it just returns a bunch of
XML and I notice that "GetUserPhoto" is not there.

Thanks,
Steve...

**Comment by tseward** [comment]
Ok, so in AD you can only add 100 KB pictures -- they're fairly small and low quality.  And sorry, GetUserPhotos which is used for picture import from Exchange, is only available on Exchange 2013 or higher. AD is going to be your only option. Did you enable the 'FoundationSync User Photos Library' on the web where you're intending to store pictures?

**Comment by whitenoiz** [comment]
Yes, I created a picture library in called "UserPhotos" and put the full URL before the /UserPhotos bit in the configuration.  If It can't pull from Exchange then how do I configure it to pull from AD?

**Comment by whitenoiz** [comment]
OK, I got it.  I just remvoed the Exchange URL and it synced everything from AD.  Sweet!
The only thing I would like to change is the file name.  Is it possible to have the file name simply match the User Name (in our case firstname.lastname, instead of the "oPqvMqVn7MbHUAsDCvdbKPa+GmM=") ?  Then we can actually use the UserPhotos library to browse and see who all the staff are, which is actually one of the board's requests that I am trying to fullfill.
